mbed_app.json

```diff
@@ -1,7 +1,6 @@
 {
     "macros": [
-        "ARM_UC_USE_PAL_BLOCKDEVICE=1",
-        "MBED_BOOTLOADER_SIZE=(32*1024)"
```

Comment:
@teetak01 Why was this macro removed? We are seeing now errors due to this removal in our CI. What we are missing now?

Reply:
It was removed by accident as part of other changes. This will impact Nucleo F429ZI compilation if using the mbed_app.json. The configs/eth_v4.json for.ex. should compile still fine.

https://github.com/ARMmbed/mbed-cloud-client-example/blob/9884def7222a5c86f1b0f8c170f3269cb70e2ab3/configs/eth_v4.json#L2-L5